examples/scripts/project/categorization/bootstrap_taxonomy.py

```diff
@@ -0,0 +1,169 @@
+"""Example script for bootstrapping a Tamr Categorization project with taxonomy categories"""
+import argparse
+import logging
+import pandas as pd
+import requests
+
+from typing import Dict, Any
+
+import tamr_toolbox as tbox
+from tamr_toolbox.project.schema_mapping import schema
+
+# Uncomment for bootstrapping categories at a specific taxonomy tier
+# from tamr_toolbox.project.categorization import metrics
+
+
+LOGGER = logging.getLogger(__name__)
+
+CATEGORY_ATTRIBUTE_NAME = "Category Name"
+FULL_PATH_LIST_NAME = "Full Path List"
+FULL_PATH_STR_NAME = "Full Path"
+COLUMN_NAMES = [FULL_PATH_STR_NAME, FULL_PATH_LIST_NAME]
+PK_NAME = "Primary Key"
+
+
+def main(*, instance_connection_info: Dict[str, Any], categorization_project_id: str) -> bool:
+    """
+    Bootstraps taxonomy in a categorization project
+
+    Args:
+        instance_connection_info: Information for connecting to Tamr (host, port, username etc)
+        categorization_project_id: The id of the target categorization project
+
+    Returns: Boolean indicating whether boostrap was successful or not
+
+    Raises:
+        TypeError: retrieved project is not a categorization project
+
+    """
+
+    # Create the tamr client
+    tamr_client = tbox.utils.client.create(**instance_connection_info)
+
+    # Retrieve the project
+    project = tamr_client.projects.by_resource_id(categorization_project_id)
+    LOGGER.info(f"Retrieved project with name: {project.name}")
+
+    # Get the project taxonomy
+    # NOTE: .taxonomy() function breaks on newer tamr instances SUP-5075
+    try:
+        project.taxonomy()
```

Comment:
Is there a workaround for this issue?

Reply:
I do not know of any workaround at the moment. :(

Reply:
Ok, I've updated the ticket. Let's see what happens there - if can't get a patch then will need to figure out a workaround for this